proto/o5/dante/v1/topic/dead_message_topic.proto

```diff
@@ -18,5 +18,5 @@ service DeadMessageTopic {
 }
 
 message DeadMessage {
-  o5.dante.v1.DeadMessage dead = 1;
+  o5.dante.v1.DeadMessageState dead = 1;
```

Comment:
I don't think the state should be sent in a message.

I actually think the fields should be inlined here, then copied to the create event in code.